rpc/api.proto

```diff
@@ -743,3 +743,53 @@ message VerifyMessageRequest {
 message VerifyMessageResponse {
 	bool valid = 1;
 }
+
+service DecodeMessageService {
+	rpc DecodeRawTransaction (DecodeRawTransactionRequest) returns (DecodeRawTransactionResponse);
+}
+
+message DecodedTransaction {
+	message ScriptSig {
```

Comment:
probably don't need this message, the script bytes and disasm can be inlined in the Input message